pkg/shell/shell.go

```diff
@@ -0,0 +1,52 @@
+package shell
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+func Get() (string, error) {
```

Comment:
have you tested `shell := os.Getenv("SHELL")` ? 
if it works for `linux`/`darwin`/`windows` it would be perfect and the code would be more readable.

Reply:
for linux and mac it works perfectly for windows it is a bit uncertain, but I guarantee with cmd if all cases fail

Reply:
We are currently not building the binary for windows yet
My question is, should we have a code that checks for windows terminals, when we know we do not even build the binaries yet?

Reply:
agreed